plugins/wazuh-core/public/services/server-security/wz-user-permissions.ts

```diff
@@ -0,0 +1,143 @@
+/*
+ * Wazuh app - React hook for get query of plugin platform searchBar
+ * Copyright (C) 2015-2022 Wazuh, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * Find more information about this on the LICENSE file.
+ */
+
+// Data extrated of /security/actions endpoint
+import wazuhPermissions from '../../../common/api-info/security-actions.json';
+
+// Constants
+const RESOURCE_ANY = '*:*:*';
+const RESOURCE_ANY_SHORT = '*:*';
+const RBAC_MODE_WHITE = 'white';
+// Utility functions
+const isAllow = (value: string) => value === 'allow';
```

Comment:
Replace with a constant instead (`allow`)

Reply:
Solved in 12187ab1638412e0cc8155aecfd00cbe75a6253f